### DISCUSSION TITLE
RichText text colors

### DISCUSSION THREAD

**Discussion by lowlighter** [payloadcms/payload#2423] [Feature Requests & Ideas]
Hi 👋 !

Would it be possible to support text colors in the rich text editor ?
(Unless it's already possible ?)

**Comment by matthijs166** [comment]
Think you can build in custom elements in the rich text editor:
https://payloadcms.com/docs/fields/rich-text#:~:text=this%20all%20works.-,Building%20custom%20elements%20and%20leaves,-You%20can%20design

This would maybe also awesome if it is added in the default elements.

You can open up a PR if you got it working
